io.c

```diff
@@ -11957,7 +11957,7 @@ io_s_foreach(VALUE v)
  *
  *  - {Open Options}[rdoc-ref:IO@Open+Options].
  *  - {Encoding options}[rdoc-ref:encodings.rdoc@Encoding+Options].
- *  - {Line Options}[rdoc-ref:IO@Line+Options].
+ *  - {Line IO}[rdoc-ref:IO@Line+IO].
```

Comment:
The `Line IO` section doesn't seem to talk about keyword arguments, the `sep` and `limit` are both positional arguments. Can we even pass these as keyword arguments?

Reply:
Out of scope for this PR?

Reply:
The Line IO section (removed in ded895baa96cb9622fcd44d308fb48341565605b) looked like this:

```
 *  === Line Options
 *
 *  A number of \IO methods accept optional keyword arguments
 *  that determine how lines in a stream are to be treated:
 *
 *  - +:chomp+: If +true+, line separators are omitted; default is +false+.
```

This is now linking to a different section that is not about keyword arguments.

Reply:
Should this simply be removed?

Reply:
Well we should keep the part about the `chomp` keyword (maybe just paste it inline), since that's actually useful information.

Reply:
Done.